.travis.yml

```diff
@@ -48,9 +47,9 @@ before_script:
 
 script:
   - set -ev
-  - mvn install -Drelease
+  - mvn clean install -Drelease
```

Comment:
Any reason to add clean before install? Keep in mind the travis always clean the machine before run the tests